app/views/budgets/ballot/_progress_bar.html.erb

```diff
@@ -2,28 +2,40 @@
   <%= @budget.formatted_heading_price(@heading) %>
 </span>
 
-<div class="progress" role="progressbar" tabindex="0"
-     id="progress"
-     aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">
-  <div class="progress-meter"
-       style="width:
-       <%= progress_bar_width(@budget.heading_price(@heading),
-                              @ballot.amount_spent(@heading)) %>">
-  </div>
-</div>
+<% if @heading.group.approval_voting? %>
```

Comment:
How about using partials to display approval or knapsack content?

Something like this:
```
<% if approval_voting? %>
  <%= render "progress_bar_approval" %>
<% else %>
  <%= render "progress_bar" %>
<% end %>
```

Reply:
Sure.